docs/shared/node-tutorial/1-code-generation.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Node Nx Tutorial - Part 1: Code Generation'
+description: In this tutorial you'll create a backend-focused workspace with Nx.
+---
+
+{% callout type="check" title="Integrated Repo" %}
+This tutorial sets up an [integrated](/concepts/integrated-vs-package-based) repo. If you prefer a [package-based repo](/concepts/integrated-vs-package-based), check out the [Core Tutorial](/getting-started/core-tutorial).
+{% /callout %}
+
+# Node Nx Tutorial - Part 1: Code Generation
+
+In this tutorial you'll create a backend-focused workspace with Nx.
+
+## Contents
+
+- [1 - Code Generation](/node-tutorial/1-code-generation)
+- [2 - Nx Graph](/node-tutorial/2-nx-graph)
+- [3 - Task Running](/node-tutorial/3-task-running)
+- [4 - Workspace Optimization](/node-tutorial/4-workspace-optimization)
+- [5 - Summary](/node-tutorial/5-summary)
+
+## Your Objective
+
+For this tutorial, your objective is to create the initial architecture for a workspace with the following requirements:
```

Comment:
The language seems formal so the sentence is longer it should be. 

The whole thing, including the bullet points, can be described something like that:

In this tutorial you will create a workspace with a Express.js microservice and a command line application sharing code.